### ISSUE TITLE
scroll element is not focusable

### ISSUE THREAD

**Issue by dinutraveltickets** [ThePacielloGroup/ARC-Toolkit#71]
The element contains overflowing content that can be reached by scrolling, however the scrolling is not accessible to keyboard users.

> The table has scrollbar. If I tab the application using key board, the nvda reader and the tab goes to that table which has no tab index and in arc run test showing error "scroll element is not focusable"

**To Reproduce**

> Identify any steps required to reproduce the behavior and/or link to a test case that demonstrates the issue.

Test Case (CodePen/JSBin/etc.):

**Expected behavior**

> A clear and concise description of what you expected to happen.

**Screenshots**

![Image](https://github.com/user-attachments/assets/76542bc9-3544-46e2-8822-e70d6c4cfc6e)

> If applicable, add screenshots to help explain your problem.

**Version information**

- Browser and version:
- ARC Toolkit version:

**Additional context**

> Add any other context about the problem here.

**Comment by poorgeek** [comment]
Hi @dinutraveltickets it isn't clear from your description what the issue is. Can you provide an example of your table?
